bin/oref0-bluetoothup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+MAC=$1
+
+if ( ifconfig wlan0 | grep -q "inet addr" ) && ( ifconfig bnep0 | grep -q "inet addr" ); then
+   sudo bt-pan client $MAC -d
+fi
+
+if ! ( hciconfig -a | grep -q "PSCAN" ) ; then
+   sudo killall bluetoothd
+   sudo /usr/local/bin/bluetoothd --experimental &
+fi
+
+if ( hciconfig -a | grep -q "DOWN" ) ; then
```

Comment:
the experimental part is the same, but one brings up the hci device while the other it's all ready up, just needs to kill the bluetooth and start it backup. The Down only happened occasionally, and I'm wondering if that's been part of the problem that people have while trying to pair.  as in doing things in the wrong order.